### ISSUE TITLE
Preview image goes blank when using custom data 

### ISSUE THREAD

**Issue by ishbaid** [microlinkhq/sdk#133]
## Bug Report
**Current Behavior**

Microlink loads with correct image / title / description for a second and then flashes and preview image disappears. 

`https://images.weserv.nl/?url=undefined 410 Error`

This only happens when using custom data. 

```
function Item(props: Props) {
  const { classes, title, width, url, description } = props;
  return (
    <div className={classes.root}>
      <div className={classes.item}>
        <MicrolinkCard
          data={{
            title,
            description
          }}
          image={["logo"]}
          url={url}
          size="normal"
          target="_blank"
          style={{ flexDirection: "column" }}
        />
      </div>
    </div>
  );
}
```

**Expected behavior/code**

Would expect preview image to be persistent

**Additional context/Screenshots**

Note: I'm using React v16.7.0-alpha.0 and the new React Hooks. 

<img width="582" alt="screen shot 2018-11-20 at 12 14 10 am" src="https://user-images.githubusercontent.com/6070130/48752849-375ebc00-ec59-11e8-9972-1f08c8438ac3.png">

**Comment by Kikobeats** [comment]
Hello,

Can you share the URL you are trying to reach? 🙂

**Comment by ishbaid** [comment]
It happens to every link I've tried, but here are the four of the links in the screenshot:

https://www.commonapp.org/

https://www.usnews.com/best-colleges

https://www.naviance.com/

https://www.niche.com/

**Comment by Kikobeats** [comment]
Hum, I tested your URLs over microlink.io website and they work fine.

Can you test it creating a fork on [codesandbox](https://codesandbox.io/s/qqr6knpvp6)?

**Comment by ishbaid** [comment]
Hmm it's working just fine on codesandbox... but still working on my local server. I wonder if it has anything to do with the newest React version or using React Hooks. Let me investigate more and report back.

**Comment by ishbaid** [comment]
This should be merged with #123 - seems most related.
